### DISCUSSION TITLE
What database does wg-easy use?

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [wg-easy/wg-easy#1128] [Q&A] [answered]
What database does wg-easy use?

**Comment by eandersons** [comment, marked as answer]
It seems that WireGuard Easy is not using any database.

**Reply by eandersons** [reply]
This has been changed since.
Yesterday the pull request #1330 that adds database ([lowdb](/typicode/lowdb)) was merged. It will be some time though till these changes will be in the production image.
